src/Artesian/GMEPublicOffers/GMEPublicOfferQuery.py

```diff
@@ -262,7 +262,7 @@ def __buildRequest(self: GMEPublicOfferQuery) -> str:
             sep = ","
             operator = sep.join(map(str, qp.operator))
```

Comment:
the name of query parameters shall match the API.

so if the paramter is 'operators', query property should be 'operators'.

Reply:
naming alligned